source/adapters/cuda/CMakeLists.txt

```diff
@@ -3,43 +3,43 @@
 # See LICENSE.TXT
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-set(CUDA_DIR "${SYCL_ADAPTER_DIR}/sycl/plugins/unified_runtime/ur/adapters/cuda" CACHE PATH "CUDA adapter directory")
+set(CUDA_DIR "${CMAKE_CURRENT_SOURCE_DIR}" CACHE PATH "CUDA adapter directory")
 
 set(TARGET_NAME ur_adapter_cuda)
 
 add_ur_adapter(${TARGET_NAME}
     SHARED
-    ${CUDA_DIR}/ur_interface_loader.cpp
-    ${CUDA_DIR}/adapter.hpp
-    ${CUDA_DIR}/adapter.cpp
-    ${CUDA_DIR}/command_buffer.hpp
-    ${CUDA_DIR}/command_buffer.cpp
-    ${CUDA_DIR}/common.hpp
-    ${CUDA_DIR}/common.cpp
-    ${CUDA_DIR}/context.hpp
-    ${CUDA_DIR}/context.cpp
-    ${CUDA_DIR}/device.hpp
-    ${CUDA_DIR}/device.cpp
-    ${CUDA_DIR}/enqueue.cpp
-    ${CUDA_DIR}/event.hpp
-    ${CUDA_DIR}/event.cpp
-    ${CUDA_DIR}/kernel.hpp
-    ${CUDA_DIR}/kernel.cpp
-    ${CUDA_DIR}/memory.hpp
-    ${CUDA_DIR}/memory.cpp
-    ${CUDA_DIR}/platform.hpp
-    ${CUDA_DIR}/platform.cpp
-    ${CUDA_DIR}/program.hpp
-    ${CUDA_DIR}/program.cpp
-    ${CUDA_DIR}/queue.hpp
-    ${CUDA_DIR}/queue.cpp
-    ${CUDA_DIR}/sampler.hpp
-    ${CUDA_DIR}/sampler.cpp
-    ${CUDA_DIR}/tracing.cpp
-    ${CUDA_DIR}/usm.cpp
-    ${CUDA_DIR}/usm_p2p.cpp
-    ${CUDA_DIR}/../../ur.cpp
-    ${CUDA_DIR}/../../ur.hpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/ur_interface_loader.cpp
```

Comment:
`image.cpp`/`image.hpp` are missing from the CMake

Reply:
Adding this caused some compile errors (from werror). To avoid changing the adapter on this PR (which makes it hard to review), I will do this change on a follow up PR.

Reply:
Ok, makes sense!